### PR TITLE
Add computeSignerListHash for computing the ledger index/id

### PR DESCRIPTION
**Pull request by sublimator**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -90,12 +99,11 @@ function computeTrustlineHash(address1, address2, currency) {
   var address1Hex = addressToHex(address1);
   var address2Hex = addressToHex(address2);
 
-  var swap = (new BigNumber(address1Hex, 16)).greaterThan(
-    new BigNumber(address2Hex, 16));
+  var swap = address1Hex > address2Hex;
```
Comment (clark800) [inline src/index.js]: This is too risky: `'1' < '02'` is `false`, `'1B' < '1a'` is `true`, etc.

**Comment by clark800** [comment]
LGTM aside from one comment

**Comment by sublimator** [comment]
Nah, base jumping is risky haha. The hex is normalised first there before
comparison, and is not random user inpurt, but I take your point. It was
senseless change.

Feel free to take over the pull request. I just whipped it together while
waiting for something to install.

**Comment by clark800** [comment]
Would you mind just reverting that one change?

**Comment by sublimator** [comment]
On mobile :)

**Comment by sublimator** [comment]
Updated

**Pull request merged**